GOVERNANCE.md

```diff
@@ -26,9 +26,8 @@ StackStorm uses a three-tiered system of Maintainer roles:
   * Have full member write access to [StackStorm](https://github.com/stackstorm/) and [StackStorm-Exchange](https://github.com/stackstorm-exchange) Github organizations, CI/CD, Moderator at [forum](https://forum.stackstorm.com/), [Slack](https://stackstorm.com/community-signup) and other Community platforms.
   * Receive **one vote** in the voting process.
 
-### Maintainer responsibilities
-* In general dedicate at least 1+ full day per week (summarized) on StackStorm development.
-* Monitor Slack, Forums and other Communication channels with StackStorm Community (delayed response is perfectly acceptable).
+### Maintainer Activities
+* Monitor one of: Slack, Forums, and other Communication channels with StackStorm Community (delayed response is perfectly acceptable).
```

Comment:
What about `one or more of` ? To me `one of` sounds like a hard limit but I guess it's in our interest to keep that up to each maintainer themselves. 

But it looks good enough to me and shouldn't prevent the PR from being merged.

Reply:
makes sense

Reply:
```suggestion
* Monitor one or more of: Slack, Forums, and other Communication channels with StackStorm Community (delayed response is perfectly acceptable).
```
Like this?